modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenOperation.java

```diff
@@ -29,9 +29,9 @@
             isArray, isMultipart,
             isResponseBinary = false, isResponseFile = false, hasReference = false,
             isRestfulIndex, isRestfulShow, isRestfulCreate, isRestfulUpdate, isRestfulDestroy,
-            isRestful, isDeprecated, isCallbackRequest, uniqueItems;
-    public String path, operationId, returnType, returnFormat, httpMethod, returnBaseType,
-            returnContainer, summary, unescapedNotes, notes, baseName, defaultResponse; 
+            isRestful, isDeprecated, isCallbackRequest, uniqueItems, returnTypeIsMetaDataResponse, returnTypeIsMetaOnlyResponse, returnTypeSupportsEntities, returnTypeIsModel, returnTypeIsArray;
```

Comment:
it kind of weird to have a concept of entity in a very generic object called CodegenOperation. I think this should be handled in the specific codegen in the java code. I don't think any of these variable should be here.

Reply:
Maybe follow the pattern here?
`class ExtendedCodegenOperation extends CodegenOperation {`